NotationV0/src/lib/credentials.ts

```diff
@@ -0,0 +1,58 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as taskLib from 'azure-pipelines-task-lib/task';
+
+export async function getVaultCredentials(): Promise<{ [key: string]: string | undefined }> {
```

Comment:
Why do we allow this function to return `undefined` again? And when will this happen?

Reply:
Good catch. Removed `undefined`